lib/client/chart.js

```diff
@@ -2,11 +2,8 @@
 
 var _ = require('lodash');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["_" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2042010440/issues/source?bid=2452374&fileBranchId=3130808#l3)